### ISSUE TITLE
Ability to limit height

### ISSUE THREAD

**Issue by dlvhdr** [omranjamal/mono-cd#2]
Currently, if there are lots of workspaces, I can't see the search input.
I would like to have the ability to limit the height of the results, also for aesthetic reasons.

**Comment by omranjamal** [comment]
That's an amazing idea. I'll get started on supporting this soon.

**Comment by omranjamal** [comment]
Hi, I changed mono-cd's behavior such that it takes up at most the number of rows available in the terminal, but if you still want to override this limit to a lower number, you can set `MONOCD_MAX_HEIGHT=6` _(or any other number)_ in your environment via your `.bashrc` or `.zshrc` file.

It's available in the `v1.2.0` release. Use the below command to upgrade.

```bash
curl -L "https://github.com/omranjamal/mono-cd/releases/latest/download/install.sh" | sh -
```

**Comment by dlvhdr** [comment]
Thank you!🙏
